deployment/sriov-network-operator/values.yaml

```diff
@@ -25,9 +25,56 @@ operator:
   nameOverride: ""
   fullnameOverride: ""
   resourcePrefix: "openshift.io"
-  enableAdmissionController: false
   cniBinPath: "/opt/cni/bin"
   clusterType: "kubernetes"
+  admissionControllers:
+    enabled: false
+    certificates:
+      secretNames:
+        operator: "operator-webhook-cert"
+        injector: "network-resources-injector-cert"
+      certManager:
+        # When enabled, makes use of certificates managed by cert-manager.
+        enabled: false
+        # When enabled, certificates are generated via cert-manager and then name will match the name of the secrets
+        # defined above
+        generateSelfSigned: false
```

Comment:
do we support non self signed ?
certificates in template point to self signed issuer

Reply:
See case 4 in the description of the PR. If a user wants to bring a certificate managed by cert-manager, they can create it outside of Helm and use that via setting the `.operator.admissionControllers.certificates.secretNames` variable. 

In general, 4 cases are supported, which are listed in the description of the PR.

Reply:
yes sorry, was a long day yesterday :)